docs/source/tutorial.rst

```diff
@@ -155,31 +191,33 @@ These can be called with :meth:`~graphix.pattern.Pattern.standardize` and :meth:
 >>> pattern.standardize()
 >>> pattern.shift_signals()
 >>> pattern.print_pattern()
-N, node = 0
-N, node = 1
 N, node = 2
 N, node = 3
 N, node = 4
 N, node = 5
 N, node = 6
 N, node = 7
-E, nodes = (6, 7)
-E, nodes = (3, 6)
-E, nodes = (5, 6)
-E, nodes = (4, 5)
-E, nodes = (1, 4)
-E, nodes = (2, 3)
 E, nodes = (0, 2)
-M, node = 0, plane = XY, angle(pi) = -0.11530492922405373, s-domain = [], t_domain = []
+E, nodes = (2, 3)
+E, nodes = (1, 4)
+E, nodes = (4, 5)
+E, nodes = (5, 6)
+E, nodes = (6, 3)
+E, nodes = (6, 7)
+M, node = 0, plane = XY, angle(pi) = -0.2975038024267561, s-domain = [], t_domain = []
 M, node = 2, plane = XY, angle(pi) = 0, s-domain = [], t_domain = []
-M, node = 1, plane = XY, angle(pi) = -0.08641619841768139, s-domain = [], t_domain = []
+M, node = 1, plane = XY, angle(pi) = -0.14788446865973076, s-domain = [], t_domain = []
 M, node = 4, plane = XY, angle(pi) = 0, s-domain = [], t_domain = []
 M, node = 5, plane = XY, angle(pi) = 0, s-domain = [4], t_domain = []
 M, node = 6, plane = XY, angle(pi) = 0, s-domain = [], t_domain = []
+X byproduct, node = 3, domain = [2]
+X byproduct, node = 7, domain = [2, 4, 6]
 Z byproduct, node = 3, domain = [0, 1, 5]
 Z byproduct, node = 7, domain = [1, 5]
-X byproduct, node = 7, domain = [2, 4, 6]
-X byproduct, node = 3, domain = [2]
+
+.. figure:: ./../imgs/pattern_visualization_3.png
+    :scale: 60 %
+    :alt: pattern visualization after standarziation
 
 The command sequence is now much clear and note that the byproduct commands now only apply to output nodes (3, 7).
```

Comment:
change this line to 
`The command sequence is now simpler and note that the most byproduct commands now apply to output nodes (3, 7).`

Reply:
added to PR